server/requirements.txt

```diff
@@ -15,4 +15,4 @@ timm==0.6.13
 typeguard==2.13.3
 accelerate==0.18.0
 pytesseract==0.3.10
-gradio==3.24.1
\ No newline at end of file
+gradio==4.19.2
```

Comment:
**Suggestion**: Ensure compatibility of the new gradio version with the project's current dependencies and overall environment. Given the significant version leap, it's crucial to test all functionalities relying on gradio to prevent runtime errors or unexpected behavior. Consider checking the gradio release notes for any breaking changes or migration steps. <br> **Code Suggestion**: 
 ```
 -gradio==3.24.1
 +gradio==4.19.2
 ```